src/pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -97,6 +97,3 @@ def optimizer_step(
 
     def state_dict(self) -> Dict[str, Any]:
         return amp.state_dict()
-
-    def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
-        amp.load_state_dict(state_dict)
```

Comment:
By deleting this, we do nothing now (super implementation) for all stages by default.